### ISSUE TITLE
can't get all message bug!!!

### ISSUE THREAD

**Issue by jackkam85** [orkestral/venom#1245]
## Description

My code runs for a month. But I couldn't receive any information since yesterday. . .
"function start" is not work
 ```

import {create} from 'venom-bot';

create(
    'abab', //Pass the name of the client you want to start the bot
    //catchQR
    (base64Qrimg, asciiQR, attempts, urlCode) => {
      console.log('Number of attempts to read the qrcode: ', attempts);
      console.log('Terminal qrcode: ', asciiQR);
      console.log('base64 image string qrcode: ', base64Qrimg);
      console.log('urlCode (data-ref): ', urlCode);
    },
    // statusFind
    (statusSession, session) => {
      console.log('Status Session: ', statusSession); //return isLogged || notLogged || browserClose || qrReadSuccess || qrReadFail || autocloseCalled || desconnectedMobile || deleteToken || chatsAvailable || deviceNotConnected || serverWssNotConnected || noOpenBrowser
      //Create session wss return "serverClose" case server for close
      console.log('Session name: ', session);
    },
    // options
    {
      folderNameToken: 'tokens', //folder name when saving tokens
      mkdirFolderToken: '', //folder directory tokens, just inside the venom folder, example:  { mkdirFolderToken: '/node_modules', } //will save the tokens folder in the node_modules directory
      headless: false, // Headless chrome
       
      debug: false, // Opens a debug session
      logQR: false, // Logs QR automatically in terminal
      
      updatesLog: true, // Logs info updates automatically in terminal
      autoClose: 0, // Automatically closes the venom-bot only when scanning the QR code (default 60 seconds, if you want to turn it off, assign 0 or false)
      createPathFileToken: false //creates a folder when inserting an object in the client's browser, to work it is necessary to pass the parameters in the function create browserSessionToken
    },
    // BrowserSessionToken
    // To receive the client's token use the function await clinet.getSessionTokenBrowser()
    undefined,
    // BrowserInstance
    (browser, waPage) => {
      console.log('Browser PID:', browser.process().pid);
      waPage.screenshot({ path: 'screenshot.png' });
    }
)
  .then((client) => start(client))
  .catch((erro) => {
    console.log('Error:');
    console.log(erro);
  });

function start(client) {
  console.log(client)
  console.log("SSS")
  client.onAnyMessage((message) => {
      console.log(message)
    if (message.body === 'Hi' && message.isGroupMsg === false) {
      client
        .sendText(message.from, 'Welcome Venom 🕷')
        .then((result) => {
          console.log('Result: ', result); //return object success
        })
        .catch((erro) => {
          console.error('Error when sending: ', erro); //return object error
        });
    }
  });
}
```

## Environment

- **Venom version(s):3.2.0
- **Browser:Chrome 87,
- **OS: all
- **Node version: node 16.

## Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

## Log Output

```
If relevant, paste all of your Log Output
```

## Your Code

```
If relevant, paste all of your challenge code in here
```

## Additional context / Screenshot

Add any other context about the problem here. If applicable, add screenshots to help explain.

**Comment by juniorsnts** [comment]
same problem here!!!

**Comment by ZerXus** [comment]
Have same problem. Dont work callback function

**Comment by rn2web** [comment]
same here... stop working.

**Comment by henriquevc** [comment]
I solved this issue by updating the venom to 3.1.12

**Comment by rn2web** [comment]
> I solved this issue by updating the venom to 3.1.12

i'm using the same version 3.2.12

**Comment by paydjo** [comment]
i've solved this problem with upgrade nodejs.
i got this problem when node v10,
after i've upgrade to node v14, my code run again

**Comment by orkestral** [comment]
fixed in version 4.0.1
